app/src/store/store.ts

```diff
@@ -134,14 +138,14 @@ const store: StoreOptions<RootState> = {
     },
 
     /**
-     *  find a post from DataService by given id
+     *  Find a post from DataService by given id
```

Comment:
They not only find the post, the post will also loaded from the server. Find sounds more like if it exist